src/index.js

```diff
@@ -117,38 +119,53 @@ $('#submit_team-member').on('click', function(event) {
   }
 })
 
-$('#submit_book-room-button').on('click', function(event) {
+// PORTAL PAGES EVENT LISTENERS
+
+$('#button_find-room-js').on('click', function(event) {
   $('.div_available-rooms').remove();
+  $('.error_no-rooms').addClass('hide');
```

Comment:
.toggle() might work for this?

Reply:
I had toggle() initial but since a few different events hide or show the message, toggle kept getting out of sync so I adjusted it back